t/full-stack.t

```diff
@@ -243,6 +244,32 @@ like(
     'test 2 is restarted by killing worker'
 );
 
+$driver->find_element_by_id('cancel_running')->click();
```

Comment:
My thought was that the web driver would still be in the same page as before (Result page for job #2, being cloned as #3), and since i wanted job #4 to be taken next, cancelling #3 would be the next thing to do. 

Obviously i was horribly wrong